nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -250,19 +250,20 @@ define([
          */
         function filterFileParameters(appSpec) {
             const fileParams = appSpec.parameters.filter((param) => {
-                if (param.text_options && param.text_options.is_output_name) {
-                    return true;
-                }
                 return (
                     param.dynamic_dropdown_options &&
                     param.dynamic_dropdown_options.data_source === 'ftp_staging'
                 );
             });
+            const outputParams = appSpec.parameters.filter((param) => {
+                return param.text_options && param.text_options.is_output_name;
+            });
```

Comment:
Ah good, I need this for the job status tab!

Reply:
Good, I'm glad it's useful outside of this part!